### ISSUE TITLE
Allow global dpi change

### ISSUE THREAD

**Issue by jkrumbiegel** [JuliaPlots/AbstractPlotting.jl#335]
I think it would be good to have a global `const Ref{Float64} dpi` or something like that, which would allow exporting plots at different bitmap sizes without changing anything but the dpi value. Right now, a lot of code assumes pixels, so if I want a plot at higher resolution, all fonts will be too small because they don't grow with larger plot size. But if all values for font sizes, line widths, etc. would be multiplied with that dpi value at some late stage in the pipeline, that would enable flexible scaling.

**Comment by asinghvi17** [comment]
Agreed, that's an issue I ran into this week trying to get a plot ready for presentation.

**Comment by SimonDanisch** [comment]
I think this shouldn't be a global, but rather a property of the scene - which you can set globally as part of the theming ;) 
Btw, the scene already carries a the current dpi of the displaying screen:
https://github.com/JuliaPlots/AbstractPlotting.jl/blob/460c7e2e40a77dc89646499923a6f09715bed95f/src/units.jl#L81
Btw, if you want to not have small fonts on large screen, the solution should be to use dpi, or DIP to specify the font size.. So moving forward, I'd put the priority on that!
